wordpress-mobile/WordPress-iOS
# Readability, Part 1: RIP WPTableViewSectionHeaderFooterView

In an effort to maintain a bit of sanity, I've divided the primary readability changes into two parts.

This first part is solely dropping our usage of `WPTableViewSectionHeaderFooterView`. The forthcoming second PR disables the `WPTableViewCell` margin hack and begins following readable margins, but requires this step first.

For this PR, note:

1. First requires reviewing, merging, and updating to WP-iOS-Shared pod `0.6.0` via PR: https://github.com/wordpress-mobile/WordPress-Shared-iOS/pull/105.
2. Since building this branch alone will result in the header/footer views being way out of alignment with the `WPTableViewCell` margin hack, we'll hold off on merging this one until we're also ready to merge https://github.com/wordpress-mobile/WordPress-iOS/pull/5764 which disables the margin hack and enforces readable margins.

To test:

1.  Review the header/footer view usage in each view controller of the changed files on iPad and iPhone.
2. Ensure that the header/footer views match the default iOS margins and sizing.

Note: This PR will build with mis-aligned cells which are instead resolved via part 2: #5756

Needs review: @frosty, let me know if I've lost my mind here 😅. Feel free to go ahead and review once you've checked out https://github.com/wordpress-mobile/WordPress-Shared-iOS/pull/103.
